### ISSUE TITLE
Delete AriaHerald and collapse into utteranceQueue

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#389]
Determined in a meeting with @mbarlow12, there is no need for `AriaHerald` anymore. `AriaHerald` is responsible for updating the inner content of an element with `aria-live`. `utteranceQueue` is responsible for managing the timing of alerts in a way that ensures an AT will read all of the alert content in full.

But `AriaHerald` is only used by `utteranceQueue`, and is only separate because  it came before  `utteranceQueue` was deemed necessary. And no sim code should use AriaHerald on its own (see #387).

We should port `AriaHerald.announcePolite` into `utteranceQueue`, remove any usages of `AriaHerald.initailize`, then delete AriaHerald.

Also, utteranceQueue is driven by phet-core/Timer. And  alert content is cleared in another delay:
```
    // after a small delay, remove this alert content from the DOM so that it cannot be found again by the virtual cursor - must occur
    // after a delay for screen reader to register the change in text content
    Timer.setTimeout( function() { liveElement.textContent = ''; }, 200 );
```

Once announcePolite is moved to utteranceQueue, we should enforce that the queue delay is larger than the clearing delay to avoid timing issues.

**Comment by mbarlow12** [comment]
@jessegreenberg before digging into this I did a quick search for usage of `AriaHerald` in our sims. It's in `Sim.js`, and its usage seems pretty straightforward to remove. Just making a note here to remove the object's usages.

**Comment by zepumph** [comment]
Personally I think the division of responsibilities makes sense, and I like having UtteranceQueue implemented as a data structure, not tied to the implementation details of how it works with the HTML. Such that that can be handled instead by another file (AriaHerald).

**Comment by jessegreenberg** [comment]
This issue is stale now that we have a separate repo just for utterance-queue. Clsoing
